packages/system/dashboard/templates/vpa.yaml

```
@@ -0,0 +1,80 @@
+  name: dashboard-internal-kubeappsapis
+  namespace: cozy-dashboard
+spec:
+  targetRef:
+    apiVersion: "apps/v1"
+    kind: Deployment
+    name: dashboard-internal-kubeappsapis
+  updatePolicy:
+    updateMode: "Auto"
+  resourcePolicy:
+    containerPolicies:
+      - containerName: kubeappsapis
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)